README.md

```diff
@@ -9,7 +9,8 @@ your Android app.
 -   [Kount integration](http://www.kount.com/fraud-detection-software)
 -   [Android Studio and Android
     SDK](http://developer.android.com/sdk/index.html)
--   Target Android SDK &gt;= 10
+-   Target Android SDK &gt;= 21 (For Releases 3.3.0 and higher)  If you need support for older Targets, use Release 3.2.0)
```

Comment:
We are still supporting Android SDK 10 as the minimum for 3.3.0.

I will update this